tracetool/version.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (C) Intel Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+
+# tool version: <major version>.<minor version>.<micro version>
+# <major version> increase the version for any new feature, major feature update
+# <minor version> increase the version for any function change (new/modify function, etc.)
+#                 reset minor version to zero when major version is incremented
+# <micro version> increase the version for other change (new definition etc.)
+#                 reset micro version to zero when minor version is incremented
+
+import os
+
+version = '2.5.1'
+# get version from external if have
+if os.path.isfile('..'+os.sep+'meson.build'):
```

Comment:
imho, I think this is bad idea. We just need to hardcode the same version here as for meson.build and confgure.ac and provide a ci job to verify that version is the same across all 3 places.

Reply:
better to detect version in code, if hardcode here, next version update need change here too. someone may forget to do that.
configure use pkg cmd to get installed libva version. is this way better?

Reply:
No. configure.ac hardcodes this as well: https://github.com/intel/libva-utils/blob/master/configure.ac#L31. That's why I talk about ci job to check version match.

I do not think that runtime detection is a good thing. It needs to be embedded.

Reply:
ok, have matched to latest libva-utils version